File: src/SPA/src/app/_core/models/user-manager/usersearchrequest.ts
export interface UserSearchRequest {
  userName: string | null;
  email: string | null;
  phoneNumber: string | null;
  fullName: string | null;
  gender: number; // 0 nam, 1 nữ
  isActive: boolean | null;
  dateOfBirth: string | null;
}

File: src/API/Dtos/UserManager/UserSearchRequest.cs
namespace API.Dtos.UserManager;

public class UserSearchRequest
{
    public string? UserName { get; set; }
    public string? Email { get; set; }
    public string? PhoneNumber { get; set; }
    public string? FullName { get; set; }
    public Gender? Gender { get; set; } // 0 nam, 1 nữ
    public bool? IsActive { get; set; }
    public string? DateOfBirth { get; set; }
}
